uplink/src/collector/tcpjson.rs

```diff
@@ -93,18 +95,23 @@ impl Bridge {
         &mut self,
         mut framed: Framed<TcpStream, LinesCodec>,
     ) -> Result<(), Error> {
-        let mut bridge_partitions = HashMap::new();
-        for (stream, config) in self.config.streams.clone() {
-            bridge_partitions.insert(
-                stream.clone(),
-                Stream::new(stream, config.topic, config.buf_size, self.data_tx.clone()),
-            );
+        let flush_period = Duration::from_secs(self.config.flush_period);
+
+        let mut bridge_partitions = HashMap::<String, (Stream<Payload>, Duration)>::new();
```

Comment:
Why did we explicitly type here? Rust can figure this out due to line 106 right?